software/winrm/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is the point creating a new module?
I see only several tests which were being moved to this new module.

The implementation of the winrm protocol is available in a separate repo.
If we want to rewrite it, we can contribute to the same old repository. Implementing OSGI could be done in the old repository as well.